src/components/Playlist/Playlist.js

```diff
@@ -26,7 +26,7 @@ export class Playlist extends React.Component {
         } = this.props;
         
         return (

```

Comment:
this component can be a function component, you dont need the `handleNameChange` method/ event handler.
you can supply `this.props.onNameChange` to the `onChange` event handler on line 33